packages/block-editor/src/components/block-list/style.scss

```diff
@@ -208,7 +208,7 @@
 
 	// Active entity spotlight.
 	&.has-active-entity:not(.is-focus-mode) {
-		opacity: 0.5;
+		//opacity: 0.5;
```

Comment:
Probably need to remove this when this PR goes out of draft state.

Reply:
Probably 😅